buildscripts/macos-package.sh

```diff
@@ -65,21 +65,21 @@ function UpdateDMG {
 	device=$(hdiutil attach -readwrite -noverify -noautoopen "builds/vaultClient.temp.dmg" | egrep '^/dev/' | sed 1q | awk '{print $1}')
 
 	# Change permissions
-	chmod -Rf go-w /Volumes/vaultClient
+	chmod -Rf go-w /Volumes/udStream
 
 	# Update the build
-	cp -af builds/vaultClient_metal.app /Volumes/vaultClient/
+	cp -af builds/vaultClient_metal.app /Volumes/udStream/
```

Comment:
`vaultClient_metal.app` won't exist (it will be `udStream.app`), it's probably best if we fix this entire script to use `udStream` and upload another secret for the `udSteam.dmg`.